ajax/libs/Vague.js/package.json

```diff
@@ -26,4 +26,5 @@
     "type": "git",
     "url": "git://github.com/GianlucaGuarini/Vague.js.git"
   }
```

Comment:
Please add a comma(,) here to pass the test.

Reply:
```
   }
```
to 
```
   },
```